cmd/root.go

```diff
@@ -54,19 +51,12 @@ func customTimeEncoder(t time.Time, enc zapcore.PrimitiveArrayEncoder) {
 	enc.AppendString(emoji + " " + t.Format(time.RFC3339) + " ")
 }
 
-// func customLevelEncoder(level zapcore.Level, enc zapcore.PrimitiveArrayEncoder) {
-// 	emoji := "\U0001F430" + " Keploy:"
-// 	enc.AppendString(emoji + " " + level.CapitalString())
-// }
-
 func newRoot() *Root {
 	return &Root{
 		subCommands: []Plugins{},
 	}
 }
 
-// Execute adds all child commands to the root command.
```

Comment:
Hi @sajdakabir, comment at line 68 and 69 is the description of execute function, so please don't remove this comments.